packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -804,6 +810,20 @@ export function TransferPanelMain({
               </ExternalLink>
             </p>
           )}
+
+          {showUSDCNotice && (
+            <p className="mt-1 text-xs font-light text-white">
+              Native USDC is coming June 8th. Bridged USDC will work on Arbitrum
```

Comment:
NIT: It's June 8th already, we might want to change this to `Native USDC is live June 8th onwards.` instead of referring to future?

Reply:
Will get clear once an ongoing thread in Slack is resolved.